pwiz_tools/Skyline/Controls/Graphs/GraphFullScan.cs

```diff
@@ -598,13 +604,125 @@ private void PopulateProperties()
                     var chromSet = stateProvider.DocumentUI.Settings.MeasuredResults.Chromatograms.FirstOrDefault(
                         chrom => chrom.ContainsFile(_msDataFileScanHelper.ScanProvider.DataFilePath));
                     spectrumProperties.ReplicateName = chromSet?.Name;
+                    if (_peaks != null && _peaks.Length > 0)

```

Comment:
(Not important)
If you feel like it, you could simplify this to:
`if (_peaks?.Length > 0)`
(there's a similar if statement down below that's doing the same with with "_precursor.Spectra")